### ISSUE TITLE
[After #4674] Can't see focus on thumbnails of second page on PDP

### ISSUE THREAD

**Issue by KateChuen** [SAP/cloud-commerce-spartacus-storefront#6576]
**Environment Details**
Spartacus: epic/accessibility branch
Browser: Chrome & Safari & Firefox
Device: mac
Happens on: http://dev-com-3.accdemo.b2c.ydev.hybris.com:4200/electronics-spa/en/USD

**Steps to Reproduce**
- Go to Product Details Page of a product that has many thumbnails, for eg: 1990255
- Tab to the thumbnail of the other "page"

**Actual Result**
It has the focus but we can't see it because it hasn't changed "page".

<img width="1195" alt="Screen Shot 2020-02-14 at 1 23 06 PM" src="https://user-images.githubusercontent.com/45409515/74557925-42250100-4f2f-11ea-9bd8-1431016dc85d.png">

**Comment by kacperknapik** [comment]
This is caused by current implementation of carousel. Will be fixed after we have a new carousel. #4674

**Comment by kacperknapik** [comment]
Duplicate of #4674
